src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Extensions/NavigationBarSettingsExtensions.cs

```diff
@@ -14,19 +12,8 @@ public static class NavigationBarSettingsExtensions
         public static IMvcBuilder AddNavigationBarSettings(this IMvcBuilder builder, IConfiguration configuration)
         {
             builder.SetDefaultNavigationSection(NavigationSection.YourCohorts);
```

Comment:
This code is fine but it would be more conventional to say:

return builder
             .SetDefaultNavigationSection(...)
             .SuppressNavigationSection(...);

This is because the MVC Builder provides a 'fluent' style (which allows methods to be chained together) and so that's what people expect to see.